text_formatter_test.go

```diff
@@ -83,6 +84,16 @@ func TestEscaping_DefaultQuoteCharacter(t *testing.T) {
 	}
 }
 
+func TestEscaping_Time(t *testing.T) {
```

Comment:
Good point! 
While doing that, I discovered that `error` is handled by `fmt.Sprint` exactly like we did manually (because of https://golang.org/src/fmt/print.go?s=6273:6320#L590). Therefore I removed the special case for `error`. 

The only thing left now is a check whether the value already is a string. We could remove that one too because `fmt.Sprint` would return the string as is. However, I don't know whether `fmt.Sprintf` or the type assertion is faster. There could be a small performance trade-off... What do you think?